node/rpc_test.go

```diff
@@ -92,11 +92,11 @@ func testGetNamespacedRequest(t *testing.T, endpointName string, assertResponseO
 				require.False(t, resp.StatusCode == http.StatusOK)
 				require.Equal(t, "application/json", resp.Header.Get("Content-Type"))
 
-				var errorMessage string
+				errorMessage := new(rpc.JSONError)
 				err := json.NewDecoder(resp.Body).Decode(&errorMessage)
```

Comment:
nit: errorMessage could be passed as is, because it is a pointer type

```suggestion
				errorMessage := new(rpc.JSONError)
				err := json.NewDecoder(resp.Body).Decode(errorMessage)
```

Reply:
Thanks! nice catch